frontend/public/components/chargeback.tsx

```diff
@@ -188,14 +189,14 @@ const DataTable = ({cols, rows, schema}: DataTableProps) => {
 
   const getUnit = (col: string) => {
     const colSchema = _.find(schema.values, {'name': col});
-    return _.get(colSchema, 'unit');
+    return _.get(colSchema, 'unit', _.isFinite(_.get(colSchema, 'value')) ? 'numeric' : null);
```

Comment:
I guess this is OK as long as we aren't converting the string to a number, but I'd like to confirm this is expected behavior and not a bug.